Add support for skipping mirroring release files (#670)
pypa/bandersnatch: docs/mirror_configuration.md

json = false
```

### release-files

The mirror release-files setting is a boolean (true/false) setting that indicates that

> the package release files should be mirrored. Defaults to `true`. When specified, you

Reviewer: ```suggestion
the package release files should be mirrored. Defaults to `true`. When this option is disabled (via setting to false), you
```

> must also specify the `root_uri` configuration. If the uri is empty, it will be set

Reviewer: ```suggestion
should also specify the `root_uri` configuration. If the uri is empty, it will be set
```

> to https://files.pythonhosted.org.

Reviewer: ```suggestion
to https://files.pythonhosted.org/.
```
